valdr-ng/package.json

```diff
@@ -8,7 +8,8 @@
     "build-valdr-ng": "ng build valdr-ng",
     "build-valrd-ng-showcase": "ng build valdr-ng-showcase",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test --browsers ChromeHeadless --watch=false",
+    "test-local": "ng test"
```

Comment:
maybe rename the one for CI - to test-ci, and leave the ng test one to just test as previously